proto/api/api.proto

```diff
@@ -198,7 +198,7 @@ message GetBlockMetaByHashRequest {
 
 message GetBlockMetasResponse {
   uint64 total = 2;
-  repeated iotextypes.BlockMeta blkMetas = 1;
+  repeated iotextypes.BlockHeaderCore blkMetas = 1;
```

Comment:
I think we may not want to change this, because core data structure and api output fields may differ